composer.json

```diff
@@ -1,34 +0,0 @@
-{
-	"name": "mediawiki/rdfio",
-	"type": "mediawiki-extension",
-	"description": "RDFIO extension for importing data in RDF format to Semantic MediaWiki",
-	"keywords": [
-		"MediaWiki",
-		"Semantic MediaWiki",
-		"RDF",
-		"Linked Data"
-	],
-	"homepage": "<attempt to pull from ExtensionName.php, else blank>",
```

Comment:
This needs to be cleaned. "<attempt to pull from ExtensionName.php, else blank>",